src/gurobi_optimods/opf/grbformulator_ac.py

```diff
@@ -769,6 +816,52 @@ def lpformulator_ac_create_constraints(alldata, model):
 
     logger.info(f"    {count} balance constraints added.")
 
+    if alldata["skipjabr"] and alldata["use_ef"]:
```

Comment:
Could you add a comment here to describe the constraints being added?

Reply:
there's one already line 820: `# voltage magnitude bounds`